test/dialects/oracle_test.py

```diff
@@ -0,0 +1,24 @@
+"""Tests specific to the oracle dialect."""
```

Comment:
Remove this file completely. We much prefer to add SQL examples to `test/fixtures/dialects/oracle`.

Create a `comment.sql` file in that directory and put the following in it:

```sql
COMMENT ON COLUMN employees.job_id 
   IS 'abbreviated job title';
```

Then run `tox -e generate-fixture-yml` which will generate a `comment.yml` file in same directory.

Add both to this branch and then both files are then automatically included in our test suite.

Managing SQL examples as test cases is MUCH easier than python code. We only revert to actual Python code for tests if there's a specific bit of Python code we want to test and can't test it with SQL statement but don;'t see that here so don't think we need this file.